python/tskit/tables.py

```diff
@@ -1733,6 +1744,147 @@ def deduplicate_sites(self):
         self.ll_tables.deduplicate_sites()
         # TODO add provenance
 
+    def slice(
+            self, start=None, stop=None, reset_coordinates=True, simplify=True,
+            record_provenance=True):
+        """
+        Returns a copy of this set of tables which include only information in
+        the genomic interval between ``start`` and ``stop``.  Edges are truncated
+        to this interval, and sites not covered by the sliced region are discarded.
```

Comment:
Should perhaps clarify that "The node and individual tables are not modified, so note that after this operation there may be nodes in the node table not ancestral to any samples (this does not affect most operations, however)."

Reply:
Never mind; this is covered by the `simplify` option below.